package.json

```diff
@@ -58,5 +60,17 @@
       "default": "mt1",
       "order": 3
     }
+  },
+  "standard": {
+    "globals": [
+      "atom",
+      "setup",
```

Comment:
Consider adding the `mocha` env instead of specifying all the mocha globals?

Reply:
@50Wliu: That sounds promising. :smile: Can you point me toward an example?

Reply:
Exactly like is done [here](https://github.com/atom/atom/blob/572aec74d9fb612849a2b02cb5e54d515341c0fd/package.json#L178-L189), but with `"mocha": true`.

Note that `mocha` is a [predefined environment](https://eslint.org/docs/user-guide/configuring#specifying-environments).

Reply:
Nice, yeah I think we should use the `mocha: true` environment.

Reply:
Thanks! db56166f6510e04df6b541642c3c108b2da21fd8 adds the `"mocha": true` env config.